CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+
+Depending on what you want to do, we're suggesting you read the respective guidelines:
+- [Asking a question](###how-to-ask-a-question)
+- [Reporting a bug](###how-to-report-a-bug)
+- [Proposing enhancements](###how-to-propose-enhancements)
+- [Submitting changes](###how-to-submit-changes)
+
+In some cases, you might want to add a new algorithm for Oríon, make sure to check out the [plugin documentation](https://orion.readthedocs.io/en/latest/plugins/base.html).
+
+### How to ask a question
+Asking a question is also a contribution to the project! We'll be happy to help you if you have any question. Before opening a new issue, make sure to do a quick search. It's possible your question has already been answered! Otherwise, go ahead. We're looking forward to help you.
+
+### How to report a bug
+You found a bug? Great! Before submitting, make sure you're experiencing the bug on the latest version of Oríon and that it's not already opened in our issue tracker.
+
+When reporting a bug, please include: 
+- A clear and descriptive title for the issue
+- Your operating system and its version
+- The version of Oríon (even if you're using the latest!)
+- A description of the bug
+- The steps to reproduce the bug
+- Extra details about your setup that might help us to reproduce the bug if not included previously
+
+### How to propose enhancements
+We're thrilled to hear you found a way to make Oríon better through minor improvements to completely new features! 
+Before creating enhancement suggestions, please check that your idea is not already present in the list of issues. You might find that you don't need to create one and can just join in the discussion directly and give your opinion!
+
+When suggesting enhancements, it is a good idea to:
+- Write a clear and descriptive title for the issue
+- Motivate why this feature is relevant regarding [Oríon's mission](https://github.com/Epistimio/orion).
+- Explain in detail how it would work.
+- Keep the scope as narrow as possible, to make it easier to implement.
+- Add as much details as possible!
+
+### How to submit changes
+We're grateful you're considering making changes to Oríon! To get started, you need to first fork the repository and then create a new branch on your fork where you'll do your changes. Once you implemented the changes, submit a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests>). All changes to Oríon are done through PRs, where they will be peer-reviewed and checked against our continuous integration system to ensure the quality of the code base.
+
+During this processs keep in mind to:
+- Set a descriptive and short branch name
+- Create a descriptive and clear title for the PR
+- Motivate the *why* and *how* of the PR (get inspiration from current or past PRs!)
+- Write [good commits](https://chris.beams.io/posts/git-commit/)
```

Comment:
Yeeeeess! :+1: :1st_place_medal: :smile:

Reply:
I realized I was using past tense all this time in my commits when this project uses imperative 😇.